smpp/receiver.go

```diff
@@ -17,6 +17,7 @@ import (
 
 // Receiver implements an SMPP client receiver.
 type Receiver struct {
+	Name                 string
```

Comment:
At this layer, it should only be a general 'data' of interface{} type. The handler and the application using it should decide what goes into it.

Reply:
Changed as suggested